src/game/behaviors/door.inc.c

```diff
@@ -65,6 +71,7 @@ void bhv_door_loop(void) {
     switch (o->oAction) {
         case DOOR_ACT_CLOSED:
             cur_obj_init_animation_with_sound(DOOR_ANIM_CLOSED);
+            load_object_collision_model();
```

Comment:
Collision is being loaded twice.

Reply:
It's not, it was moved not duplicated.

Reply:
I'm an idiot, I'm looking at the star door behavior